index.html

```diff
@@ -1,300 +1,444 @@
-              <img src="asset/images/contactform.png" alt="contact">
-        <aside class="touch">
-        <h2 class="last">Contact me</h2>
-        <p class="end">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
-          I’d love to help with it!</p>
-        
-          <form class="form1" action="https://formspree.io/f/xnqkndvo" method="post">
-            <label for="name"></label>
-            <input type="text" id="name" name="name" placeholder="Lorem Ipsum"
-            required maxlength="30"><br>
-  
-            <label for="email1"></label>
-            <input type="email" id="email1" placeholder="loremipsum@gmail.com" name="email" required><br>
-            
-            <lebe for="message"></lebe>
-            <textarea name="message" id="message" placeholder="Write your message here..." size="500" maxlength="500" required></textarea><br>
-  
-            <button class="submit" type="submit">Get in touch</button>
-          </form>
-        </aside>
-        </article>
+        <!-- ------------  Facebook 360  ------------ -->
+        <section class="tonic" id="facebook">
+          <aside class="desk">
+            <div class="nature">
+              <img
+                src="image/Snapshoot Portfo.jpg"
+                alt="Facebook"
+                width="100%"
+                height="100%"
+              />
+            </div>
+            <div class="nature1">
+            <img
+                src="image/tonic.yoga.png"
+                alt="nature"
+                width="100%"
+                height="100%"
+              />
+            </div>
+          </aside>
+          <aside class="desk">
+            <div class="tonic2 tonic4">
+              <h2 class="title">Facebook 360</h2>
+              <div class="canopy">
+                <h6>FACEBOOK</h6>
+                <img
+                  src="image/Counter.jpg"
+                  alt="conter"
+                  width="8px"
+                  height="8px"
+                />
+                <p class="back">Full Stack Dev</p>
+                <img
+                  src="image/Counter.jpg"
+                  alt="conter"
+                  width="8px"
+                  height="8px"
+                />
+                <p class="back">2015</p>
+              </div>
+              <p class="daily">
+                Exploring the future of media in Facebook's first Virtual
+                Reality app; a place to discover and enjoy 360 photos and videos
+                on Gear VR.
+              </p>
+              <ul class="study">
+                <li class="box1 box">html</li>
+                <li class="box4 box">Ruby on rails</li>
+                <li class="box2 box">css</li>
+                <li class="box3 box">javaScript</li>
+              </ul>
+              <button class="button">See project</button>
+            </div>
+          </aside>
         </section>
```

Comment:
- [ ] As suggested by the previous reviewer, when you add the s and run your code locally, there will still be lots of changes to be made.  Please make the changes to all images url affected in this project.

One issue I noticed in your portfolio project is that you are referencing a different image URL than the one you have in your project folder. For example, in your HTML file, there should be an assets folder before you write the image folder, and the image folder is spelled images with an s, yours has no s. The example below uses your code but with the correct URL:


```.html
<section class="tonic" id="facebook">
          <aside class="desk">
            <div class="nature">
              <img
                src="asset/images/ Portfo.jpg"
                alt="Facebook"
                width="100%"
                height="100%"
              />
            </div>
            <div class="nature1">
            <img
                src="asset/images/tonic.yoga.png"
                alt="nature"
                width="100%"
                height="100%"
              />
            </div>
          </aside>
          <aside class="desk">
            <div class="tonic2 tonic4">
              <h2 class="title">Facebook 360</h2>
              <div class="canopy">
                <h6>FACEBOOK</h6>
                <img
                  src="asset/images/Counter.jpg"
                  alt="conter"
                  width="8px"
                  height="8px"
                />
                <p class="back">Full Stack Dev</p>
                <img
                  src="asset/images/Counter.jpg"
                  alt="conter"
                  width="8px"
                  height="8px"
                />
                <p class="back">2015</p>
              </div>
              <p class="daily">
                Exploring the future of media in Facebook's first Virtual
                Reality app; a place to discover and enjoy 360 photos and videos
                on Gear VR.
              </p>
              <ul class="study">
                <li class="box1 box">html</li>
                <li class="box4 box">Ruby on rails</li>
                <li class="box2 box">css</li>
                <li class="box3 box">javaScript</li>
              </ul>
              <button class="button">See project</button>
            </div>
          </aside>
        </section>
```